pkg/monitoring/hyperconverged/rules/alerts/operator_alerts.go

```diff
@@ -12,8 +12,10 @@ const (
 	installationNotCompletedAlert = "HCOInstallationIncomplete"
 	singleStackIPv6Alert          = "SingleStackIPv6Unsupported"
 	MisconfiguredDeschedulerAlert = "HCOMisconfiguredDescheduler"
-	severityAlertLabelKey         = "severity"
-	healthImpactAlertLabelKey     = "operator_health_impact"
+	unsupportedArchitecturesAlert = "HCODICTWithNoSupportedArchitecture"
```

Comment:
@nunnatsa  I recommend `HCOUnsupportedCustomGoldenImage` instead. From my POV it reflects the meaning, the impact in a clearer way.